.npmignore

```diff
@@ -0,0 +1,2 @@
+*
+!dist/*
```

Comment:
As I understand of this pull request you will use approach - publish to npm with built code. 
Have you checked architecture capability (like when you are building on ```x86``` or ```x64``` machine and try to use it on raspberry pi (arm architecture))?

PS: I ask this question because in case of ready build on ```x86``` and use it on ```arm``` we may have problems with mongoose and work with mongo, but it's need to check.